README.md

```diff
@@ -32,8 +32,8 @@ You can join the server below and join in #support under the **Wumpcord** catego
 ```js
 const { Client } = require('wumpcord');
 const client = new Client({
-  token: '',
-  ws: { intents: ['guilds', 'guildMessages'] }
+  intents: ['guilds', 'guildMessages'],

```

Comment:
unrelated to this PR but I think intents should be in SCREAMING_SNAKE_CASE, so in this case GUILDS and GUILD_MESSAGES to indicate that they're constant strings and also for parity with the discord api

Reply:
Since `feat/stage` and Wumpcord are gonna be updated as a major release, I can do that. 👍

Reply:
Honestly, yeah. 

I thought of letting them put them in lowercase and just convert them to uppercase but someone can just import the Intents anyways so, yeah. +1